internal/provider/datasource_directory.go

```diff
@@ -35,7 +35,7 @@ func (ds *directoryDataSource) Configure(_ context.Context, req datasource.Confi
 
 func (ds *directoryDataSource) Schema(_ context.Context, _ datasource.SchemaRequest, resp *datasource.SchemaResponse) {
 	resp.Schema = schema.Schema{
-		MarkdownDescription: `Get the details about a directory.
+		MarkdownDescription: `Gets the details about a directory.
```

Comment:
Looking at the other provider's documentation, they use 3rd person singular for the data-source description.